docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -30,88 +30,3 @@ This limit will be applied to every container of the given component.
 +
 For `plugin` components, RAM limits can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-
-. Specify container memory request for components
-+
-To specify a container(s) memory request for `plugin` use the `memoryRequest` parameter:
-+
-.Specify container memory request for components
-====
-[source,yaml]
-----
-  components:
-   - name: exec-plugin
-     plugin:
-      id: eclipse/machine-exec-plugin/0.0.1
-      memoryLimit: 1Gi
-      memoryRequest: 512M
-   - name: maven
-     container:
-      image: eclipe/maven-jdk8:latest
-      memoryLimit: 512M
-      memoryRequest: 256M
-----
-====
-+
-This limit will be applied to every container of the given component.
-+
-For `plugin` components, RAM requests can be described in the plug-in descriptor file, typically named `meta.yaml`.
-+
-If they are not specified, the values are undetermined: they may or may not be inferred from the application that consumes the devfile or from Kubernetes.
-+
-
-. Specify container CPU limit for components
-+
-To specify a container(s) CPU limit for `plugin` or `container` use the `cpuLimit` parameter:
-+
-.Specify container CPU limit for components
-====
-[source,yaml]
-----
-  components:
-   - name: exec-plugin
```

Comment:
Although cpu limit stays, plugin will be removed from the devfile spec (devfile/api#333) in the future. We can remove the plugin part of this example and leave the container part.